automation-tools/rdopkg.md

```diff
@@ -0,0 +1,62 @@
+# RDOPKG
+
+- `rdopkg` is an RPM packaging automation tool.
+- [ :computer: github.com/softwarefactory-project/rdopkgo](https://github.com/softwarefactory-project/rdopkg), [ :scroll: documentation](https://github.com/softwarefactory-project/rdopkg/blob/master/doc/rdopkg.1.adoc)
+- ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/rdopkg.svg), [![PyPI](https://img.shields.io/pypi/v/rdopkg.svg)](https://pypi.org/project/rdopkg/), [ :package: fedora packages](https://src.fedoraproject.org/rpms/rdopkg)
+
+:heavy_plus_sign:
+- cloning package distgit and setting up remotes
+- introducing patches
+- rebases to new versions
+- sending changes for review
+- modifying .spec file: bumping versions, managing patches, writing changelog, producing meaningful commit messages, …
+- persistance (`rdopkg --continue`) -- The state is stored in a file named .rdopkg.json in the current directory.
+
+:heavy_minus_sign:
+- .
```

Comment:
does it work outside of dist-git?

Reply:
I am not sure if I got your question -- it has 4 remote repositories:

- upstream git
- rdo distgit
- patched git repo
- metadata repo